charts/fluid-dataloader/jindofsx/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+### 0.9.0
+- Support for setting affinity,tolerations,nodeSelector,schedulerName
```

Comment:
Suggest to use `0.6.0` or `0.5.1` to indicate the Chart changes

Reply:
done